SecuritySystemUWP/SecuritySystemUWP/TelemetryHelper.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+
+namespace SecuritySystemUWP
+{
+    /// <summary>
+    /// Helper class for app insights telemetry events.
+    /// Allows tracking page views, exceptions and other telemetry through the Microsoft Application Insights service.
+    /// </summary>
+    internal static class TelemetryHelper
+    {
+        // app insights telemetry object
+        internal static Microsoft.ApplicationInsights.TelemetryClient _TelemetryClient = new TelemetryClient();
+
+        /// <summary>
+        /// Helper private method to add additional common properties to all telemetry events
+        /// </summary>
+        /// <param name="properties">common global telemetry properties to log</param>
+        private static void AddCommonProperties(ref IDictionary<string, string> properties)
+        {
+            
+            properties.Add("Custom_AppVersion", EnvironmentSettings.GetAppVersion());
+            properties.Add("Custom_OSVersion", EnvironmentSettings.GetOSVersion());
+
+#if MS_INTERNAL_ONLY // Do not send this app insights telemetry data for external customers. Microsoft only.
+            properties.Add("userAlias", App.Controller.XmlSettings.MicrosoftAlias);
+            properties.Add("Custom_DeviceName", EnvironmentSettings.GetDeviceName());
+            properties.Add("Custom_IPAddress", EnvironmentSettings.GetIPAddress());
+#endif
+        }
+
+        /// <summary>
+        /// Log and track custom app insights event with global common properities
+        /// See https://azure.microsoft.com/en-us/documentation/articles/app-insights-api-custom-events-metrics/#api-summary
+        /// </summary>
+        /// <param name="eventName"></param>
+        internal static void TrackEvent(string eventName)
+        {
+            Dictionary<string, string> properties = new Dictionary<string, string>();
+            // call helper method to add common properties
```

Comment:
Did you intend to actually make the call to the "addCommonProperties" method here? This comment is in a few places